src/openrct2/world/Footpath.cpp

```diff
@@ -1596,6 +1596,11 @@ void FootpathUpdatePathWideFlags(const CoordsXY& footpathPos)
     // FootpathClearWide(x, y);
     // y -= 0x20;
 
+    // Only consider approx. 1/4 of tiles for wide path status
```

Comment:
Are you sure about these fractions? I would have expected something like 1/7 here.

Reply:
0xE0 is 224, so technically it's 224/1000 tiles. I was rounding off; I think it's meant to approximate 25%, yeah.

Reply:
Argh no, it's hexadecimal, so (0xFF - 0xE0) / 0xFF ~= 0.12156, so much closer to 12.5%. You're right.

Reply:
I think that may be off by one - there 256 (0x100) different coordinates in a byte, from 0x0 to 0xFF, so I think the calculation is (0x100 - 0xE0) / 0x100, which neatly translates to 256.

Just for comparison, if you’d compare it with 0xFF instead of 0xE0, you’d still hit 0xFF once. In my calculation that’s (0x100 - 0xFF) / 0x100, = 1/256 whereas that would be 0 in your calculation.